extract_pov_edits/ingest_utils/process.py

```diff
@@ -149,3 +197,38 @@ def process(content, cur_sents):
        error = True
   return diff(sents, cur_sents, content['rev_id']), error
 
+
+def matched(sent1, sent2):
```

Comment:
is this here twice?

Reply:
No, but the same function was implemented in another file as well.